commands/cli/parse.go

```diff
@@ -416,7 +417,7 @@ func appendFile(fpath string, argDef *cmds.Argument, recursive, hidden bool) (fi
 		}
 	}
 
-	return files.NewSerialFile(path.Base(fpath), fpath, hidden, stat)
+	return files.NewSerialFile(path.Base(filepath.ToSlash(fpath)), fpath, hidden, stat)
```

Comment:
instead of doing this here, we should on line 401 do:
```go
fpath = filepath.Clean(fpath)
fpath, err = filepath.EvalSymlinks(fpath)
...
fpath = filepath.ToSlash(fpath)
```